csrank/metrics.py

```diff
@@ -123,3 +121,15 @@ def topk_acc(y_true, y_pred):
         return acc
 
     return topk_acc
+
+def dcg_at_k(r, k): #required for LambdaMART
```

Comment:
The comment can be removed here, if this is supposed to be a general purpose function.
Better would be a docstring, documenting what the function is computing.